test/integration/space-members-integration.js

```diff
@@ -0,0 +1,19 @@
+export default function spaceMembersTests (t, space) {
+  t.test('Gets spaceMembers', t => {
+    t.plan(3)
+    return space.getSpaceMembers().then(response => {
+      t.ok(response.sys, 'sys')
+      t.ok(response.sys.type, 'Array')
+      t.ok(response.items, 'items')
+    })
+  })
+  t.test('Gets spaceMember', t => {
+    t.plan(4)
+    return space.getSpaceMember('0PCYk22mt1xD7gTKZhHycN').then(response => {
+      t.ok(response.sys, 'sys')
+      t.ok(response.sys.type, 'SpaceMember')
```

Comment:
I think you would be able to use `t.equal` in this tests.

Reply:
oh thanks that sounds like a good idea!